README.md

```diff
@@ -20,14 +20,28 @@ Add the following to your `WORKSPACE` file to add the necessary external depende
 
 ```python
 git_repository(
+    name = "io_bazel_rules_docker",
+    commit = "{HEAD}",
```

Comment:
Why are we using {HEAD} here? This won't work when people copy/paste.

Reply:
I added this when going through the OSS process because I didn't have a commit yet.  The "can't copy/paste" is somewhat by design for now, I think that once we cut a first release we should update this to reference releases.